### PR TITLE
Refactor landing page

### PR DESCRIPTION
**Pull request by ColemanMitch**
Cleaning up the landing page (see video below 👇)

### CODE REVIEW

**Comment by ColemanMitch** [comment]
https://user-images.githubusercontent.com/44789534/210636992-7b36146e-f61d-4ece-9e9a-0e651120b673.mov

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "web-vitals": "^1.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```
Comment (dwilliams27) [inline package.json:29]: What dis for?

**Review by dwilliams27 — APPROVED** [review]
Changing the fucking game attaching a VIDEO to a PR lol

**Comment by ColemanMitch** [comment]
> Changing the fucking game attaching a VIDEO to a PR lol

brother,,, GitHub is I N C R E D I B L E  with what it can do

File: package.json (hunk 1 above)
Comment (ColemanMitch) [reply package.json:29]: my `npm start` commands fails if I don't change this start script for whatever reason, might just be a me thing 🤷

Comment (ColemanMitch) [reply package.json:29]: https://stackoverflow.com/a/69713899

Comment (ColemanMitch) [reply package.json:29]: ⬆️ this is the SO Q&A I followed

**Comment by ColemanMitch** [comment]
changes are live if you'd like to see for yourself: https://spowerfy.onrender.com/ 👀 @dwilliams27

**Pull request merged**